src/morphology/german/detectAndStemRegularParticiple.js

```diff
@@ -0,0 +1,196 @@
+import exceptionsParticiplesActive from "../../researches/german/passiveVoice/exceptionsParticiplesActive";
+import { exceptions } from "../../researches/german/passiveVoice/regex";
+
+/**
+ * Detects whether a word is a regular participle without a prefix and if so, returns the stem.
+ *
+ * @param {Object}  morphologyDataVerbs The German morphology data for verbs.
+ * @param {string}  word                The word (not stemmed) to check.
+ *
+ * @returns {string}    The stem.
+ */
+const detectAndStemParticiplesWithoutPrefixes = function( morphologyDataVerbs, word ) {
+	const participleRegex1 = new RegExp( "^" + morphologyDataVerbs.participleRegexes.participleRegexGeStemTEnd );
```

Comment:
As discussed, I'm not not implementing this change since the regexes make use of the morphology data, which is only available within the scope of the function.